src/app/confirm-dialog/confirm-dialog.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import {MatDialog, MatDialogRef, MAT_DIALOG_DATA} from '@angular/material/dialog';
+
+@Component({
+  selector: 'app-confirm-dialog',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7556402)
```suggestion
  selector: "app-confirm-dialog",
```